index.js

```diff
@@ -395,21 +405,17 @@ module.exports = async function(options) {
               extend: 'apostrophe-pieces',
               name: 'site',
               beforeConstruct: function(self, options) {
-                options.addFields = [
-                  {
-                    type: 'array',
-                    name: 'hostnamesArray',
-                    label: 'Hostnames',
-                    schema: [
-                      {
-                        type: 'string',
-                        required: true,
-                        name: 'hostname',
-                        help: 'All valid hostnames for the site must be on this list, for instance both example.com and www.example.com'
-                      }
-                    ]
-                  }
-                ].concat(options.addFields || []);
+                options.addFields = [{
```

Comment:
This is not an improvement in readability, did this come from an autoformatter? Please submit PRs without unnecessary whitespace and formatting changes. Thanks!

Reply:
(It's different if you feel those changes are the whole point, but not in a PR about something else if possible)